<ui version="4.0" >
 <author>Andrey Rijov</author>
 <class>Settings_Window</class>
 <widget class="QDialog" name="Settings_Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AQEMU Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../usr/kde/3.5/share/icons/crystalsvg/16x16/actions/view_choose.png</normaloff>../../../../usr/kde/3.5/share/icons/crystalsvg/16x16/actions/view_choose.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="Label_Default_VM_Template" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Default VM Template:</string>
       </property>
       <property name="buddy" >
        <cstring>CB_Default_VM_Template</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_Default_VM_Template" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Create_Template_from_VM" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Create Template...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="Label_VM_Folder" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>AQEMU VM Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit_VM_Folder" />
     </item>
     <item>
      <widget class="QToolButton" name="TB_VM_Folder" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="Label_Language" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>AQEMU Interface Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="CB_Language" >
       <item>
        <property name="text" >
         <string>English</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="Line" name="line_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QCheckBox" name="CH_Use_Device_Manager" >
       <property name="text" >
        <string>Use Device Manager by Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CH_Use_USB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Support USB Host Devices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CH_Show_Emulator_Control_Window" >
       <property name="text" >
        <string>Show Emulator Control Window</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CH_Use_VNC_Display" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Use Embedded Display in Emulator Control Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CH_Screenshot_for_OS_Logo" >
       <property name="text" >
        <string>Use Screenshot for OS Icon in Save Mode</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CH_64_Icons" >
       <property name="text" >
        <string>Use 64x64 Screenshot Icons</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QLabel" name="Label_Other_Settings" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Other Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" >
    <widget class="QPushButton" name="Button_Advanced_Settings" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Advanced Settings...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QPushButton" name="Button_Screenshots_Settings" >
     <property name="text" >
      <string>Screenshots Settings...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QPushButton" name="Button_QEMU_Binary" >
     <property name="text" >
      <string>Manage QEMU Binary...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="QDialogButtonBox" name="Button_Box" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CB_Default_VM_Template</tabstop>
  <tabstop>Button_Create_Template_from_VM</tabstop>
  <tabstop>Edit_VM_Folder</tabstop>
  <tabstop>TB_VM_Folder</tabstop>
  <tabstop>CB_Language</tabstop>
  <tabstop>CH_Use_Device_Manager</tabstop>
  <tabstop>CH_Use_USB</tabstop>
  <tabstop>CH_Show_Emulator_Control_Window</tabstop>
  <tabstop>CH_Use_VNC_Display</tabstop>
  <tabstop>CH_Screenshot_for_OS_Logo</tabstop>
  <tabstop>CH_64_Icons</tabstop>
  <tabstop>Button_Advanced_Settings</tabstop>
  <tabstop>Button_Screenshots_Settings</tabstop>
  <tabstop>Button_QEMU_Binary</tabstop>
  <tabstop>Button_Box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
